be/src/column/datum.h

```diff
@@ -188,9 +188,16 @@ class Datum {
         return std::visit([&](const auto& arg) { return is_equal(arg); }, key);
     }
 
+    bool operator==(const Datum& rhs) const {
+        return _value == rhs._value;
```

Comment:
It may not work for **RefType**, such as `BitmapValue*`, `JsonValue*` ?

Reply:
emmm.... it's use for test, I will remove it